packages/modelviewer.dev/styles/docs.css

```diff
@@ -190,3 +172,35 @@ hr.solid {
 .bolden {
   font-weight: bold;
 }
+
+/* Smaller screen/ mobile */
+@media only screen and (max-width: 800px) {
+  :root {
+    --sidebar-width: 0;
+  }
+
+  .sidebar {
+    width: var(--sidebar-width);
+  }
+
+  .outer-content {
+    margin-left: 0;
+    max-width: 100%;
+  }
+
+  .value-table {
+    width: 100%;
```

Comment:
We need some form of this sidebar toggle on the example pages since right now on mobile once you go to the examples you can't get back. Maybe just put the same sticky header on them?

Reply:
yep! in the works...